doc/usage.md

```diff
@@ -221,11 +221,11 @@ The share can be mounted in the guest with `mount -t virtiofs vfkitTag /mnt`, wi
 The `--device virtio-gpu` option allows the user to add graphical devices to the virtual machine.
 
 #### Arguments
-- `height`: the vertical resolution of the graphical device's resolution. Defaults to 800
-- `width`: the horizontal resolution of the graphical device's resolution. Defaults to 600
+- `height`: the vertical resolution of the graphical device's resolution. Defaults to 600
+- `width`: the horizontal resolution of the graphical device's resolution. Defaults to 800
```

Comment:
Could you switch the order of height and width in most places so that width comes before height? Seeing 600 then 800 to mean 800x600 will confuse me every time ^^